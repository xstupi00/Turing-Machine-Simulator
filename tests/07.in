S a K R
S a A L
A   S R
A   B L
B   A R
K b S L
K b Q R
Q c M R
Q c K L
M   K d
K d F a
abc
