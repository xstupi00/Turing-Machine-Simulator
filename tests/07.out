Sabc
aKbc
abQc
abcM 
abcKd
abcFa
